stock_critical_forecast_link_replenish/tests/TEST_INSTRUCTIONS.rst

```diff
@@ -0,0 +1,8 @@
+- Open the Inventory app
+- Create the replenishment rule for a product with a critical order date (delivery), set the minimum and maximum quantities
```

Comment:
You misunderstood. Tell which product from the demo data it is by its actual name.